harness/determined/launch/autodeepspeed.py

```diff
@@ -19,6 +19,8 @@
 from determined.common import api
 from determined.common.api import certs
 
+hostfile_path = "/tmp/hostfile.txt"
```

Comment:
slight nit: maybe just pass this into `main()` from from the `if __name__ == "__main__"` clause?  This shouldn't really need to be overwritten as part of the test.

Reply:
feels like this belongs more as a constant in the class? rather than having to pass in the same path from a test.

Reply:
I guess I'd assume you'd pass in a different value than `/tmp/hostfile.txt`.  But I guess `/tmp` is basically always writable, so maybe it doesn't matter at all.